EditorExtensions/Resources/server/services/srv-scss.js

```diff
@@ -35,8 +35,9 @@ var handleSass = function(writer, params) {
             writer.end();
         }
 
-        var css = result.css.toString();
-        var map = result.map.toString();
+        var map, css = result.css.toString();
+        if (!options.omitSourceMapUrl)
+            map = result.map.toString();
```

Comment:
Like I said earlier, the compiler services (which support sourcemap, exception being LiveScript), should return source map to Web Essentials in all cases. WE then uses source-maps for editor extensibility features.
If sourcemap in autoprefixer is conditional, then that is a (separate) bug.
If sourcemap in RTLCss is conditional, then that is by design (because rtlcss result are returned separately).
Therefore, the *real* issue here is that the post-processed source-maps should overwrite those generated by node-based compiler, but it should respect the user-intended `params.sourceMapURL` option. And hence, the solution I described above would fix it. If you are still unconvinced, you can checkout the changes in srv-scss, as this is slightly out of scope for this PR. I will take care of it once the WE solution starts building again.